docs/extending/rules.md

```diff
@@ -52,6 +52,74 @@ Rules can accept other rules as parameters:
 It is also possible to use inline references to registered evaluators without declaring rules,
 in case you do not need providing extra parameters, or chaining multiple rules together.
 
+```json
```

Comment:
Can you also adapt the docs in `extensions.schema.json`?

Reply:
I can update the schema as part of [ACS-8694](https://hyland.atlassian.net/browse/ACS-8693), since this PR is already merged, and I have to make changes near the same code there anyway. Please see commits [here](https://github.com/Alfresco/alfresco-content-app/commit/ca12e227159ddc9dc0d31248b51a2f800d6b2a0a) and [here](https://github.com/Alfresco/alfresco-content-app/commit/2c7571f49a0359ad578c099599cc68b8287433d2)

Reply:
Both looks good, thank you!